input.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+
+from vector import Vector
+
+
+def handle_key_input(user_input):
+    """Use tdl's user input features to react to keyborad input.
+
+    Returns a dictionary containing an action key and a value to be parsed by the game logic.
+    """
+    key_char = user_input.char
+
+    # Vertical and horizontal movement
+    if user_input.key == 'UP' or key_char == 'k':
+        return {'move': Vector(0, -1)}
+    elif user_input.key == 'DOWN' or key_char == 'j':
+        return {'move': Vector(0, 1)}
+    elif user_input.key == 'LEFT' or key_char == 'h':
+        return {'move': Vector(-1, 0)}
+    elif user_input.key == 'RIGHT' or key_char == 'l':
+        return {'move': Vector(1, 0)}
+
+    # Diagonal movement
```

Comment:
It's what most roguelikes use. Some also allow numpad use, which could be more intuitive to someone new to roguelikes. We could do that instead, because I'm not sure there's a way to check simultaneous key press in tdl.